core/src/locktower.rs

```diff
@@ -162,8 +162,19 @@ impl Locktower {
                 };
                 Self::update_ancestor_lockouts(&mut stake_lockouts, &vote, ancestors);
             }
-            // each account hash a stake for all the forks in the active tree for this bank
-            Self::update_ancestor_stakes(&mut stake_lockouts, bank_slot, lamports, ancestors);
+
+            if vote_state.votes.len() > 1 {
```

Comment:
Is the empty check necessary since nth returns an option?

Reply:
nope not necessary, removed in the latest